scripts/Create-Share.ps1

```diff
@@ -46,7 +46,7 @@ try {
 
     $CreateSharePs={
         $ErrorActionPreference = "Stop"
-        New-SmbShare -Name $Using:ShareName -Path $Using:Path -FullAccess $FullAccessUser
+        New-SmbShare -Name $Using:ShareName -Path $Using:Path -FullAccess $Using:FullAccessUser
```

Comment:
Can we also make the $FullAccessUser declaration in line 29 a string array variable. This allow us to pass multiple users if needed. This can be done by changing it to:
`     [string][]$FullAccessUser='everyone'`